index.html

```diff
@@ -6981,12 +7081,14 @@ <h3 id="Mortality">Mortality</h3>
                <td>Summon The Dead
                </td>
                <td>
-                  Suffer Heavy damage and choose 3 points within 50ft. Summon an undead creature on each of 
-                  these points - see the appendix for a relevant stat block. Each of these creatures 
-                  are fully controlled by you and may each immediately take their turn upon being summoned. 
-                  In future rounds they will take turns immediately before yours. On your turn you may
-                  spend 0 AP to destroy any of these creatures. You may have a maximum of 3 undead creatures 
-                  summoned at any time, and they will persist until you take a short or long rest.
+                  <strong>Raise an undead horde to fight for you.</strong>
```

Comment:
One reason why this was so strong is that they couldn't be taken out by AOE.  Making making them close together would be a slight nerf?  Not a phrasing thing tho

Reply:
We can make it a much smaller AoE, I don't mind that - question is how small? Needs to be 10ft range to make it so they can't still be very far away from eachother?